dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/dxf2/events/TrackedEntityInstanceServiceTest.java

```diff
@@ -603,6 +607,45 @@ void testSavePerson() {
     assertEquals(ImportStatus.SUCCESS, importSummary.getStatus());
   }
 
+  @Test
+  void shouldFailWhenRegisteringPersonOutsideCaptureScope() {
+
```

Comment:
Usually, we don't include a line break between the method name and its code, and we format the test to follow the Arrange-Act-Assert pattern.